index.html

```diff
@@ -0,0 +1,392 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" />
+    <link rel="stylesheet" href="style.css">
+    <title>KMarketer</title>
+</head>
+
+<body>
+
+    <header class="header">
+        <div class="header__nav">
+            <h1 class="header__logo">KMarketer</h1>
+            <div class="header__lists">
+                <div class="header__link-text">
+                    <ul class="header-items">
+                        <li class="header-link"><a class="header-text" href="#">Home</a></li>
+                        <li class="header-link"><a class="header-text" href="#">Product</a></li>
+                        <li class="header-link"><a class="header-text" href="#">Pricing</a></li>
+                        <li class="header-link"><a class="header-text" href="#">Contact</a></li>
+                        <li class="header-link"><a class="header-items__icon" href="#"><i
+                                    class="fa-solid fa-magnifying-glass"></i></a></li>
+                        <li class="header-link"><a class="header-items__icon" href="#"><i
+                                    class="fa-solid fa-cart-shopping"></i></a>
+                    </ul>
+                </div>
+
+            </div>
+        </div>
+
+        <div class="header__content">
+            <h1 class="title title__header">Experts are here solve <br>
+                your business problem.</h1>
+            <h4 class="text text__description">We know how large objects will act, but things on a <br>
+                small scale just do not act that way.</h4>
+            <div class="header__buttons">
+                <ul class="header-items header-items--gap">
```

Comment:
`header-items`  should have been element of `header` and you've made it a block 
may be use `header__items`